resources/minikube.rb

```diff
@@ -0,0 +1,264 @@
+resource_name :minikube
+
+property :version, String, default: ''
+property :path, String, default: '/usr/local/bin'
+property :k8s_version, String, default: ''
+property :network_plugin, String, default: ''
+property :bootstrapper, String, default: ''
+property :vm_driver, String, default: 'none'
+property :user_name, String, default: ''
+
+default_action :install
+
+load_current_value do
+end
+
+action :install do
+  version = new_resource.version
+  version = latest_ver if new_resource.version.empty?
+
+  raise 'version is nil' if version.nil?
+
+  current = current_ver
+
+  if current != version
+
+    remove_minikube unless current.nil?
+
+    get_minikube_url = "https://storage.googleapis.com/minikube/releases/#{version}/minikube-#{sys}-#{arch}"
+
+    directory new_resource.path do
+      mode 0755
+      action :create
+      not_if { Dir.exist?(new_resource.path) }
+    end
+
+    execute 'get minikube' do
+      cwd new_resource.path
+      command "curl -Lo minikube #{get_minikube_url} && chmod +x minikube"
+      action :run
+      not_if 'which minikube'
+    end
+
+    ruby_block "check minikube exist in #{new_resource.path}" do
+      block do
+        unless ::File.exist?(binary_path)
+          Chef::Log.fatal("Could not find minikube at #{new_resource.path}!")
+        end
+      end
+      action :run
+    end
+  end
+end
+
+action :run do
```

Comment:
:run should not be introduced, we should have ":install", ":remove" action, that's enough for us.

Reply:
when minikube is stopped manually, :install should make it run again (this case is useful when I want to stop minikube temporarily and use it later)